ethereum-types/src/hash.rs

```diff
@@ -1,22 +1,22 @@
-use {U64, U128, U256, U512};
+use crate::{U256, U512};
```

Comment:
`parity-ethereum` uses both `U64` and `U128` currently. Do you have a plan for replacing them?

Reply:
I checked some of the dependent crates on crates.io and couldn't find a crate there that uses them - but maybe just bad luck. To me the usecase for having a `U64` is interesting. What does it bring to the table what a bare metal `u64` doesn't?

I did not plan to invest much more time into refactoring dependent crates.
Maybe we should revert these removals again if they lead to disadvantages.

Reply:
reverted removals